### DISCUSSION TITLE
[Feature Request] Windows Vista/7 style switcher (3D Aero Flip)

### DISCUSSION THREAD

**Discussion by XuanChen1234** [valinet/ExplorerPatcher#1669] [Ideas]
Is it Possible to integrate the Flip3D Design for App Switcher?
If it is not planned yet, I hope this will be a thing in the next update.
![3D Aero Flip](https://github.com/valinet/ExplorerPatcher/assets/140480110/5e738df7-2b76-4ae1-aaf6-15ed2ebabab1)
https://github.com/valinet/ExplorerPatcher/discussions/1381

**Comment by XuanChen1234** [comment]
@valinet Any more planned?

**Reply by valinet** [reply]
Stop tagging me in every post!

**Reply by vippium** [reply]
These guys are not looking for currently running bugs on latest beta build and preview build... Like action center overlap in preview builds and Broken Win10 taskbar on latest beta build.

They are only looking for new features in EP. Let valinet first fix currently running bugs with beta guys as it was 23H2 and let him finish his work.... I think people will understand.
Be calm valinet bro.

**Comment by vippium** [comment]
😂😂

**Comment by pyrates999** [comment]
@XuanChen1234 please close all of your feature requests that you added under issues.  Feature requests are not supposed to be under issues.
